cmd/gen-manifests/main.go

```diff
@@ -375,11 +376,15 @@ func mockDepsolve(packageSets map[string][]rpmmd.PackageSet, repos []rpmmd.RepoC
 		for _, pkgSet := range pkgSetChain {
 			for _, pkgName := range pkgSet.Include {
 				checksum := fmt.Sprintf("%x", sha256.Sum256([]byte(pkgName)))
+				// generate predictable but non-empty
+				// release/version numbers
+				ver := strconv.Itoa(int(pkgName[0]) % 9)
+				rel := strconv.Itoa(int(pkgName[1]) % 9)
 				spec := rpmmd.PackageSpec{
 					Name:           pkgName,
 					Epoch:          0,
-					Version:        "0",
-					Release:        "0",
+					Version:        ver,
+					Release:        "r" + rel,
```

Comment:
Where does the `r` in the release come from here? All the release fields I've seen are usually a single number :)

Reply:
I may be missing something and happy to tweak as needed, I was simply looking at:
```
$ dnf info coreutils
Name         : coreutils
Version      : 9.4
Release      : 7.fc40
...
```
or am I looking at the wrong "releas" field here?

Reply:
It's the right release field, but to @supakeen's point, I don't see an `r`.

Reply:
Sure, sorry, the example release field is `7.fc40` AFAICT, on centos9 it's `36.el9` so maybe `rel + ".fk1" (or the full fake1)? I was mostly trying to make it distinct from pure numbers as it seems to be a combination of number and distro shorthand in general. Ideas welcome of course!

Reply:
Agreed.  Doesn't really matter in the end, as long as it's stable an predictable.

Reply:
Ack.